lua/aerial/init.lua

```diff
@@ -113,6 +113,13 @@ local commands = {
       desc = "Close the aerial nav window.",
     },
   },
+  {
+    cmd = "AerialTSClear",
+    func = "treesitter_clear_query_cache",
+    defn = {
+      desc = "Clear aerial's tree-sitter query cache",
+    },
+  },
```

Comment:
It's probably quite uncommon for a user to need to do this, so I'd prefer to not add another command. There's already too many commands; I'd like to shift more of those to API-only functionality, the only thing stopping me is backwards-compatibility.

Reply:
Makes sense. I removed the command, leaving only an API method exposed.